LICENSE.md

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012-2022 Scott Chacon and others
```

Comment:
This is misleading. I wrote the majority of this, so if we name any individual it should probably be me.

Generally I'm more fond of listing an org like "XboxDev Maintainers" (although that adds some legal ambiguity) or *all* contributors explicitly.

Reply:
Total oversight on my part, I copied the license text from a sample on GH. Fixed!

I went with XboxDev maintainers, as I am dubious that the maintenance overhead of keeping an explicit list of contributors is worthwhile and using "and others" seems just as ambiguous as the blanket maintainers. Happy to change if you'd prefer.

Reply:
Looks okay now; the README still mentions "All Rights Reserved" I believe - that'd be incorrect now.

Reply:
Fixed, and lowercased "maintainers" in `LICENSE.md` to align with use in `README`